index.css

```
@@ -325,7 +333,9 @@ button.button > * {
 
     opacity: 1;
 
-    transition: top 0.5s, opacity 0.5s;
+    transition:
+        top 0.5s,
+        opacity 0.5s;
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)